### ISSUE TITLE
Estimated training result and things affect result

### ISSUE THREAD

**Issue by TWDH** [xingyizhou/CenterNet#598]
@xingyizhou Hello dalao, I want to train my model based on the original CenterNet, but I don't have 8 GPUs. Will the number of GPUs and Learning rate(I set to 1.25e-4 initially) affect the training result? 

Would anybody what's the estimated loss(loss, hm_loss,wh_loss,off_loss) after training 140? 
I am training the centernet, dla backone, on COCO dataset.
(python main.py ctdet --exp_id coco_dla --batch_size 8 --master_batch 1 --lr 1.25e-4  --gpus 0 --num_workers 0)

Many thanks!

**Comment by xingyizhou** [comment]
It will, and will take forever to train on COCO. I would recommend you to work on Pascal VOC first if you only have 1 gpu. I don't have the losses with me, and I have never looked at the loss values. It turns out the loss value does not concur well with the AP in CenterNet.

**Comment by TWDH** [comment]
Hi @xingyizhou , thanks very much! I was trying to download the PASCAL VOC Annotations in COCO Format via the provided website, but unable to download it (access denied) Any alternative way to download it?

This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Access denied.</Message>
<Details>
Anonymous caller does not have storage.objects.get access to coco-dataset/external/PASCAL_VOC.zip.
</Details>
</Error>

**Comment by TWDH** [comment]
@xingyizhou Hello, I must come to you again, the link:https://storage.googleapis.com/coco-dataset/external/PASCAL_VOC.zip seems doesn't work anymore. Is there any alternative link we can use? I tried to convert the voc data myself but there's always some unexpected error when trying to test it. Thank you so so much!

**Comment by TWDH** [comment]
I have created coco format coco annotation myself and works fine. Here is the link:https://drive.google.com/drive/folders/1KtfQovwqE1XD4yh0aBtShrlRnhrDhLrX?usp=sharing

Thanks
